Add inline with SNAPPY_ATTRIBUTE_ALWAYS_INLINE on AdvanceToNextTag to fix:
1097 |         size_t tag_type = AdvanceToNextTag(&ip, &tag);
     |                           ~~~~~~~~~~~~~~~~^~~~~~~~~~~
../snappy.cc:1017:8: error: inlining failed in call to 'always_inline'
'size_t snappy::AdvanceToNextTag(const uint8_t**, size_t*)': function body can be overwritten at link time

--- snappy.cc	2021-05-05 02:53:34.000000000 +0400
+++ snappy.cc	2021-05-24 01:24:59.124654893 +0400
@@ -1017 +1017 @@ SNAPPY_ATTRIBUTE_ALWAYS_INLINE
-size_t AdvanceToNextTag(const uint8_t** ip_p, size_t* tag) {
+inline size_t AdvanceToNextTag(const uint8_t** ip_p, size_t* tag) {
